gordian/gordian.py

```diff
@@ -144,8 +156,15 @@ def apply_transformations(args, transformations, pr_created_callback=None):
     config = Config(args.config_file)
     transform(args, transformations, config.get_data(), pr_created_callback=pr_created_callback)
 
+def get_pr_description(args):
+    if args.description_file is not None:
+        with open(args.description_file,'r') as fh:
+            return fh.read()
+    return args.description
+
 def transform(args, transformations, repositories, pr_created_callback):
     pull_request_urls = []
+    pr_description = get_pr_description(args)
```

Comment:
I'd like to keep the interface of this transform as agnostic as possible, I think adding an extra `description` argument with '' as default value would be ideal.
Then in the apply_transformation I'd add the get_pr_description() call

```python
def apply_transformations(args, transformations, pr_created_callback=None):
    config = Config(args.config_file)	    config = Config(args.config_file)
    transform(args, transformations, config.get_data(), pr_created_callback=pr_created_callback, pr_description=get_pr_description(args))

def get_pr_description(args):
    if args.description_file is not None:
        with open(args.description_file,'r') as fh:
            return fh.read()
    return args.description

def transform(args, transformations, repositories, pr_created_callback, description=''):
    ...
```